tests/database/test_db.py

```
@@ -0,0 +1,1273 @@
+
+        with patch("time.sleep"):
+            result = send_submit_query(session)
+
+        assert mock_connection.execute.call_count == 3
+        mock_connection.execute.assert_any_call("ROLLBACK")
+        mock_connection.execute.assert_any_call("COMMIT")
+        assert result
+
+    @patch("nettacker.database.db.messages", return_value="mocked fail message")
+    @patch("nettacker.database.db.logger.warn")
+    def test_send_submit_query_sqlite_failure(self, mock_warn, mock_messages):
```

Ruff (0.14.10)

Unused method argument: `mock_messages`

(ARG002)